### PR TITLE
Edit syncs with edit sync wizard

### PR DESCRIPTION
**Pull request by yeldah3**
https://www.loom.com/share/d32d0dd5582e4908b6624ab7ba65bf11

Screenshot 2024-05-24 at 2 35 09 PM

### CODE REVIEW

**Comment by utD9FeifNY6fLo35PfwlcznLtzQdMpBpXqBFeZ2** [comment]
<p><a href="https://linear.app/getcensus/issue/EMBED-443/add-edit-sync-functionality">EMBED-443 Add edit sync functionality</a></p>

**Comment by yeldah3** [comment]
* **#43** <a href="https://app.graphite.dev/github/pr/sutrolabs/census-embedded-demo/43?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#39** <a href="https://app.graphite.dev/github/pr/sutrolabs/census-embedded-demo/39?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @yeldah3 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by UOyXPJKtDj7BSXHeMVTcWvKtBkMK1h0hv7lAjh0** [comment]
### <span aria-hidden="true">✅</span> Deploy Preview for *powered-by-census-demo* ready!


|  Name | Link |
|:-:|------------------------|
|<span aria-hidden="true">🔨</span> Latest commit | e69f43576f5579c2f9f374bf53e67028d1ba5aa0 |
|<span aria-hidden="true">🔍</span> Latest deploy log | https://app.netlify.com/sites/powered-by-census-demo/deploys/66510bd36689c90008f70b41 |
|<span aria-hidden="true">😎</span> Deploy Preview | [https://deploy-preview-39--powered-by-census-demo.netlify.app](https://deploy-preview-39--powered-by-census-demo.netlify.app) |
|<span aria-hidden="true">📱</span> Preview on mobile | <details><summary> Toggle QR Code... </summary><br /><br />QR Code<br /><br />_Use your smartphone camera to open QR code link._</details> |
---

_To edit notification comments on pull requests, go to your [Netlify site configuration](https://app.netlify.com/sites/powered-by-census-demo/configuration/deploys#deploy-notifications)._

File: components/SyncObject.js [hunk 1]
```diff
@@ -103,109 +134,148 @@ export function SyncObject({
...
-              {
-                ["paused"]: !sync.paused,
-              },
-              null,
-              2,
-            )}
-          </pre>
-        }
-      />
+      {!editSyncWizardLink && (
```
Comment (ZIxOozkggyhzlkZUTwcKB8W32wDEYU8dARxlISY) [inline components/SyncObject.js]: were these randomly popping up when moving the mouse around if were in edit sync wizard mode? Why only show them if `!editSyncWizardLink`?

**Review by LemonFace0309 — APPROVED** [review]
LET'S GOOOO

File: components/SyncObject.js (hunk 1 above)
Comment (yeldah3) [reply components/SyncObject.js]: Great point! I'm not sure if this check is needed. Mainly we hide the delete/edit/pause/run buttons when showing the sync wizard, so I wasn't sure if the tooltips would be mad about their anchors being hidden. I'll check to see if anything's actually broken without this check

Comment (yeldah3) [reply components/SyncObject.js]: You right, this check does not seem to be needed 👍 yeeting!

Comment (ZIxOozkggyhzlkZUTwcKB8W32wDEYU8dARxlISY) [reply components/SyncObject.js]: woohoo!

**Pull request merged**